app/src/main/java/com/project/mobility/view/activities/OnboardingActivity.java

```diff
@@ -110,6 +113,11 @@ public void goToNextPage() {
         viewPager.setCurrentItem(currentPage + 1, true);
     }
 
+    @OnClick(R.id.onboarding_back_button)
```

Comment:
You can rename this method as navigateBackward for example and call goToPreviousPage() in it ,because what you did does not allow you to go to previous page inside onBackPressed(), when going back with the back button